Userland/Libraries/LibC/mallocdefs.h

```diff
@@ -14,7 +14,7 @@
 #define MALLOC_SCRUB_BYTE 0xdc
 #define FREE_SCRUB_BYTE 0xed
 
-static constexpr unsigned short size_classes[] = { 8, 16, 32, 64, 128, 256, 500, 1016, 2032, 4088, 8184, 16376, 32752, 0 };
+static constexpr unsigned short size_classes[] = { 8, 16, 32, 64, 128, 256, 504, 1016, 2032, 4088, 8184, 16376, 32752, 0 };
```

Comment:
Good catch! Since these are compile time constants, maybe we could add a `static_assert` to prevent this from happening again?